demo/TreeViewExamples.jsx

```diff
@@ -53,9 +53,9 @@ export function TreeViewExamples() {
   return (
     <TreeView
       nodes={nodes}
-      template={(item, index, parentItem) => (
+      template={(item, index, parentItem, relativePath) => (
         <em>
-          {item.attrA} (index: {index} parent: {parentItem?.attrA || <em>None</em>})
+          {item.attrA} (index: {index} parent: {parentItem?.attrA || <em>None</em>})<p>Path: {relativePath}</p>
```

Comment:
as the tree grows, only de index and the father element are not enough to handle changes in deep nodes. 
With this new argument, change node values is a lot easier.
![image](https://user-images.githubusercontent.com/29656850/115025067-47653f00-9e97-11eb-84df-b16f18bb8d3d.png)